src/ExperimentalMasonry/Masonry.js

```diff
@@ -32,7 +32,7 @@ type Props<T> = {|
   flexible?: boolean,
   gutterWidth?: number,
   items: Array<T>,
-  measurementStore?: Cache<T, *>,
+  measurementStore: Cache<T, *>,
```

Comment:
FYI, I opted not to make the same fix for the control Masonry since that should be going away soon